study.md

```diff
@@ -43,7 +44,8 @@ Although both Array and Enumerable have a `count` method, Array also defines the
 method `length`.  Why is `length` sensibly defined on Array but not Enumerable?
 
 ```md
-<!-- your answer here -->
+An array has fixed size, 'length' is only work for an enumerable has fixed number like array.
+'Count' method is able to take a blog which is work for a unfinity ennumerable.
```

Comment:
`length` makes sense on `Array` because arrays have a finite length.  It does not make sense for an `Enumerable` to have a `length` because it might not be finite.  For example, an enumerable that returns a digit of π on each iteration does not have a finite length.